wolox-books/src/scss/navbar.scss

```diff
@@ -0,0 +1,26 @@
+$nav-height: 50px;
+
+.navbar {
```

Comment:
I think this is just for the `header` element in the `authComponent`. Or will you use it in another elements?